spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -267,6 +288,46 @@ class Xcode
             @generator.project.targets.find { |t| t.name == 'CoconutLib-iOS' }.isa.should == 'PBXAggregateTarget'
           end
 
+          it 'creates and links app host with an iOS test native target' do
+            @generator.generate!
+            @generator.project.targets.count.should == 19
```

Comment:
nit: can remove this assertion. Its superseded by prior tests and also does not help that much. (I know it was a left over from the transferred test)